helix-core/src/test/java/org/apache/helix/controller/rebalancer/waged/model/TestClusterContext.java

```diff
@@ -68,6 +68,8 @@ public void testNormalUsage() throws IOException {
         .addPartitionToFaultZone(_testFaultZoneId, replica.getResourceName(),
             replica.getPartitionName()));
     Assert.assertEquals(context.getAssignmentForFaultZoneMap(), expectedFaultZoneMap);
+    Assert.assertEquals(context.getEstimatedMaxUtilization(), 16.0 / 20.0, 0.005);
```

Comment:
nit, please roughly explain the numbers here. They are not necessarily clear to the other readers.